charts/jaeger/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.16.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
-version: 0.18.5
+version: 0.18.6
```

Comment:
nit: but this adds a new feature so a minor bump would be appropriate

Reply:
bumped to 0.19.0